### PR TITLE
Add remaining doctests for high level functions

### PR DESCRIPTION
**Pull request by djalova**
---

## Checklist:

<details open>

<summary></summary>

- Does this pull request close an issue? We encourage you to open an issue first if this pull request (PR) is not a
  minor change.
  - [ ] No
    - [ ] The change in this pull request is minor.
  - [x] Yes
    - Closes issue in separate repo

For the following questions, only check the boxes that are applicable.

- [x] Ensure that the pull request text above the horizontal line is descriptive.
- [x] Add/update relevant tests.
- [x] Add/update relevant documents.

- Do you have triage permission of this repository?
  - [ ] No
    - You are good.
  - [x] Yes
    - Does this pull request close an issue?
      - [ ] No, this pull request also acts as an issue by itself.
        - [ ] Assign yourself.
        - [ ] Label this pull request.
        - [ ] Put a milestone.
        - [ ] Put this pull request under "In Progress" or "Under Review" pipeline on ZenHub.
        - [ ] Put this pull request under the appropriate epic on ZenHub.
        - [ ] Put an estimate on ZenHub.
        - [ ] Add dependency relationship with other issues/pull requests on ZenHub.
      - [ ] Yes, this pull request addresses an issue and does not act as an issue.
        - [ ] Connect this pull request to the underlying issue on ZenHub.
        - [ ] DON'T:
          - DON'T assign yourself or anyone else.
          - DON'T label.
          - DON'T put a milestone.
          - DON'T put this pull request under any epic on ZenHub.
          - DON'T put an estimate on ZenHub.
    - [x] When the pull request is ready, request review.
</details>

### CODE REVIEW

**Comment by djalova** [comment]
I didn't include a test for `init()` since it doesn't return anything.

**Comment by xuhdev** [comment]
Could you reopen this using a branch in this repo? The test isn't running

**Comment by djalova** [comment]
Oh right I forgot

File: pydax/_high_level.py [hunk 1]
```diff
@@ -250,7 +250,17 @@ def get_dataset_metadata(name: str, *,
 
 
 def export_schemata() -> SchemaManager:
-    "Return a copy of the ``SchemaManager`` object managed by high-level functions."
+    """Return a copy of the ``SchemaManager`` object managed by high-level functions.
+
+    :return: Return a copy of the SchemaManager object
```
Comment (xuhdev) [inline pydax/_high_level.py:255]:
```suggestion
    """Return a copy of the :class:`SchemaManager` object managed by high-level functions.

    :return: A copy of the :class:`SchemaManager` object managed by high-level functions.
```

File: pydax/_high_level.py [hunk 2]
```diff
@@ -250,7 +250,17 @@ def get_dataset_metadata(name: str, *,
...
+
+    Example:
+
+    >>> schema_manager = export_schemata()
+    >>> schema_manager.schemata
+    {'datasets': ..., 'formats': ..., 'licenses':...}
+
```
Comment (xuhdev) [inline pydax/_high_level.py:262]: Blank line

**Pull request closed without merging**